manim/scene/scene.py

```diff
@@ -763,7 +793,7 @@ def compile_animations(self, *args, **kwargs):
 
         return animations
 
-    def _get_animation_time_progression(self, animations, duration):
+    def _get_animation_time_progression(self, animations, duration, speedinfo):
```

Comment:
It would be good to replace `speedinfo` (in general, not only here) with something more explicit such that the naming of the newly introduced attributes is a bit more uniform. How about `animation_speed`?